packages/odata-service-writer/test/unit/annotations.test.ts

```diff
@@ -34,7 +34,7 @@ describe('metadata parsing', () => {
     it('getAnnotationNamespaces: metadata parsing', () => {
         expect(() => {
             getAnnotationNamespaces({ metadata: invalidEdmx });
-        }).toThrow(t('error.unparseableXML'));
+        }).toThrow(t('error.unparseableXML', { error: "Error: boolean attribute '<Schema' is not allowed.:13:5" }));
```

Comment:
This error will have to be continously updated as it refers specifically to a loc. Probably should not check that detail.